packages/keyv/src/index.js

```diff
@@ -57,6 +57,9 @@ class Keyv extends EventEmitter {
 					? iterator(this.opts.store.namespace)
 					: iterator) {
 					const data = typeof raw === 'string' ? this.opts.deserialize(raw) : raw;
+					if (this.opts.store.namespace && !key.includes(this.opts.store.namespace)) {
+						continue;
```

Comment:
we are testing this line in `@Keyv/test-suite` but in `Keyv` project we are using SQLite store and we are handling same namespace keys in query so this line is not getting tested.